src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

```diff
@@ -38,4 +38,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     <CentralPackageVersionsFileImported>true</CentralPackageVersionsFileImported>
   </PropertyGroup>
 
+  <!--
+    Setup default NuGet property values.
+    Visual Studio's project property page requires defaults to be set to inform customers what the default values are.
+  -->
+  <PropertyGroup>
+    <NuGetAudit Condition=" '$(NuGetAudit)' == '' ">true</NuGetAudit>
+    <NuGetAuditLevel Condition=" '$(NuGetAuditLevel)' == '' ">low</NuGetAuditLevel>
```

Comment:
I was chatting with @drewnoakes about this a month ago, and I agree that this is a little confusing without a description/instructions.  Should we take this opportunity to also change the property name to `NuGetAuditMinSeverity`? This will make it more intuitive.  If `NuGetAuditLevel` is set, we'll use that value, for backwards compatibility.

@JonDouglas what do you think?

Reply:
I don't think we change the name until it is GA and people give us their feedback. I do think the name proposed here could be helpful, but I also don't think we have any data to justify another property name today. We can re-visit that conversation later in my opinion. Docs and property page tooltips should be able to help this in the time being.